src/v/cloud_storage/tests/remote_partition_fuzz_test.cc

```diff
@@ -372,19 +372,24 @@ FIXTURE_TEST(
 namespace {
 
 ss::future<> scan_until_close(
-  remote_partition& partition,
-  const storage::log_reader_config& reader_config,
+  ss::shared_ptr<remote_partition> partition,
+  storage::log_reader_config reader_config,
   ss::gate& g) {
+    test_log.info("starting scan_until_close");
+    auto _ = ss::defer([] { test_log.info("exiting scan_until_close"); });
     auto guard = g.hold();
+    auto counter = size_t{0};
     while (!g.is_closed()) {
         try {
-            auto translating_reader = co_await partition.make_reader(
+            test_log.info("running scan loop nr {}", counter++);
+            auto translating_reader = co_await partition->make_reader(
               reader_config);
             auto reader = std::move(translating_reader.reader);
             auto headers_read = co_await reader.consume(
               test_consumer(), model::no_timeout);
+            test_log.info("done scan loop", counter++);
```

Comment:
is this missing formatting specifier?

```suggestion
            test_log.info("done scan loop {}", counter++);
```

Reply:
It is also incremented twice per loop. Once in `test_log.info("running scan loop nr {}", counter++);` and then here again.

Reply:
🤦 i was just talking how we should push to update seastar/log.hh to catch formatting errors